src/Pages/SignIn/Components/Button.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+function Button(props) {
+  const { text, onClick } = props;
```

Comment:
```suggestion
function Button({ text, onClick }) {
```
- props 객체 미리 구조 분해 할당 할 수 있습니다! 전부 확인해주세요-

Reply:
넵!! 이 부분 수정했습니다.